gnmi_server/server_test.go

```diff
@@ -2403,20 +2412,23 @@ func runTestSubscribe(t *testing.T, namespace string) {
 			c := client.New()
 			defer c.Close()
 			var gotNoti []client.Notification
+			var mu_gotNoti sync.Mutex
```

Comment:
Please rename this variable

https://google.github.io/styleguide/go/decisions#indent-error-flow:~:text=areas%20within%20naming.-,Underscores,-Names%20in%20Go